src/components/ebay-progress-bar-expressive/marko-tag.json

```diff
@@ -0,0 +1,20 @@
+{
+    "attribute-groups": ["html-attributes"],
+    "@*": {
+        "targetProperty": null,
+        "type": "expression"
+    },
+    "@html-attributes": "expression",
+    "@messages <message>[]": {
+        "@*": {
+            "targetProperty": null,
+            "type": "expression"
+        },
+        "@text": "string",
```

Comment:
Lets remove this. 
Unfortunately for marko 4 apps, we need to keep this file in sync with our storybook attributes otherwise it won't work properly.